app.rb

```diff
@@ -31,19 +32,74 @@
 config_overrides[:port] = ENV['APRUVE_PORT'] unless ENV['APRUVE_PORT'].nil?
 Apruve.configure(ENV['APRUVE_API_KEY'], apruve_environment, config_overrides)
 merchant_id = ENV['APRUVE_MERCHANT_ID']
+corporate_account_id = ENV['APRUVE_CORPORATE_ACCOUNT_ID']
 
 before '/webhook_notify' do
   request.body.rewind
   @webhook_body = request.body.read
 end
 
+get '/offline_order' do
+  erb :offline_orders
+end
+
+get '/customers' do
+  content_type :json
+  send_file File.expand_path('customers.json', settings.public_folder)
+end
+
+get '/shopper_id/:email' do
+  content_type :json
+  begin
+    corporate_account = Apruve::CorporateAccount.find(merchant_id, params['email'])
+    return { shoper_id: corporate_account.customer_uuid } .to_json
+  rescue Apruve::NotFound
+    status 404
+    return ''
+  end
+end
+
+post '/orders' do
+  request.body.rewind
+  payload = JSON.parse request.body.read
+  total_amount_cents = payload['orders'].map { |order|
+    order['subtotal'].to_f
+  }.reduce(:+)
+  total_amount_cents = ((total_amount_cents + payload['shipping'].to_f) * 100).round
```

Comment:
Adding this would be no problem. I guess the question is whether we want that to be another input field like shipping or whether to just use a fixed percentage?